github/teams.go

```diff
@@ -476,3 +476,88 @@ func (s *TeamsService) RemoveTeamProject(ctx context.Context, teamID int64, proj
 
 	return s.client.Do(ctx, req, nil)
 }
+
+// GroupObject reprents list of Idp Group.
+type GroupObject struct {
+	Groups []*Group `json:"groups,omitempty"`
+}
+
+// Group represents an Idp group status.
+type Group struct {
```

Comment:
Please change this to: `// IDPGroup represents an external identity provider (IDP) group.`